src/models/eventmodel.h

```diff
@@ -71,9 +71,13 @@ class EventModel : public QAbstractItemModel
         QString name;
     };
 
+public:
+    void addToFavourites(const QModelIndex& index);
+
 private:
     Data::EventResults m_data;
     QVector<Process> m_processes;
+    QVector<QModelIndex> m_favourites;
```

Comment:
I find it somewhat unfortunate to use model indices here. seemingly easy, but also deceptively wrong.

when we favorite the index `(0, 0, someParent)` and then later request the favorite for that index, but at a different column, we need to translate to keep the column request working